ironfish/src/wallet/wallet.ts

```diff
@@ -1822,14 +1757,16 @@ export class Wallet {
     return latestHead ? latestHead.hash : null
   }
 
-  async isAccountUpToDate(account: Account): Promise<boolean> {
+  async isAccountUpToDate(account: Account, confirmations?: number): Promise<boolean> {
```

Comment:
That's a good question... seems like the main case might be when you export, then re-import an account, you might want to make sure the account doesn't have a balance before deleting it, in case you haven't backed it up.

Or if you somehow create an account without a birthday, give someone the public address, they send you tokens, and you delete the account (thinking the transaction didn't work) before fully syncing up.

We could probably re-think how we prompt users about that though 🤔